File: Interview/867/transpose.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        ret=[]
        for i in range(len(matrix[0])):
            cur=[]
            for j in range(len(matrix)):
                cur.append(matrix[j][i])
            ret.append(cur)
        return ret

File: Coding/867/transpose.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        vector<vector<int>> ret;
        for(int i = 0; i < matrix[0].size();i++){
            vector<int> v;
            for(int j = 0; j<matrix.size();j++){
                v.emplace_back(matrix[j][i]);
            }
            ret.emplace_back(v);
        }
        return ret;
    }
};
